cli/dir_checksum/tests/integration_tests.rs

```diff
@@ -0,0 +1,178 @@
+extern crate tempdir;
+
+use std::{
+    fs::{self, File},
+    path::PathBuf,
+};
+
+use dir_checksum::*;
+use tempdir::TempDir;
+
+fn create_tmp_dir_with_file(dir_name: &'static str) -> anyhow::Result<(TempDir, PathBuf)> {
+    let tmp_dir = TempDir::new(dir_name)?;
+    let file_path = tmp_dir.path().join("file1.txt");
+    fs::write(&file_path, "Initial text")?;
+
+    Ok((tmp_dir, file_path))
+}
+
+#[test]
+fn hash_combinations_file_identical() -> anyhow::Result<()> {
+    let (tmp_dir, _) = create_tmp_dir_with_file("comb_file_identical")?;
+
+    let original_hash = calc_combined_checksum(tmp_dir.path())?;
+
+    assert_eq!(
+        original_hash,
+        calc_combined_checksum(tmp_dir.path())?,
+        "Hashes of the same directory must be identical"
+    );
```

Comment:
I don't get this test. Just calling the function twice and comparing the results does not seem very interesting.

Reply:
The point of this test, that if we call the function on the same dir without any changes then I should provide the same hash. But I can remove it since it's included in the next test `hash_combinations_add_then_remove_file()`